vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -977,7 +978,9 @@ class RoomDetailFragment @Inject constructor(
 
     @SuppressLint("SetTextI18n")
```

Comment:
Not related to this PR, but I think this can be removed...

Reply:
Done